glabels/Preview.cpp

```diff
@@ -101,9 +105,53 @@ namespace glabels
 			drawPaper( mModel->tmplate()->pageWidth(), mModel->tmplate()->pageHeight() );
 			drawLabels();
 			drawPreviewOverlay();
+			drawLabelNumberOverlay();
 		}
 	}
 
+	void Preview::drawLabelNumberOverlaySingle(const model::Distance& x, const model::Distance& y, const QPainterPath& path, int32_t labelInstance)
+	{
+		QBrush brush( labelNumberColor );
+
+		model::Frame *frame = mModel->tmplate()->frames().first();
+
+		model::Distance w = frame->w();
+		model::Distance h = frame->h();
+
+		model::Distance minWH = min( w, h );
+
+		char labelText[32];
+		snprintf(labelText, sizeof(labelText), "%d", labelInstance);
```

Comment:
Wouldn't this be better implemented using `QString`? E.g.:

`auto labelText = QString::number(labelInstance);`

Reply:
@sur5r yep, thats much simpler, corrected that and removed the rotation support and re-pushed for re-review

Reply:
Looks good!